### ISSUE TITLE
Add extra field values to ajax request on remote validation.

### ISSUE THREAD

**Issue by jonnywilliamson** [proengsoft/laravel-jsvalidation#88]
Sorry! I have one more question/issue about remote validation.

Take a form with 2 fields.

```html
<!DOCTYPE html>
<body>
{!! Form::open(['route'=>'processRegistration', 'id' => 'tgregister', 'role' => 'form']) !!}
<input type="text" name="telegramId" value="{{ old('telegramId') }}">
<input type="text" name="verCode" value="{{ old('verCode') }}">
<button type="submit">Submit</button>
{!! Form::close() !!}

{!! JsValidator::formRequest('App\Http\Requests\RegisterTelegramId', '#tgregister') !!}
</body>
</html>
```

`telegramId` is a simple REQUIRED rule.
`verCode` is a CUSTOM validation. Lets call it `md5Check`.

The `md5Check` is simple. It takes the `telegramId` field and md5 hashes the input and then compares the first 4 characters with what the user entered in the  `verCode` input box.

#### This is working perfecting when I have remote validating DISABLED. #####

When I enable remote validation and enter a value in the 'verCode' input box, an ajax request is fired off that contains only 2 bits of info
```
verCode:abcd
_jsvalidation:verCode
```

My validation can not be checked in this case, because the value of `telegramId` is never sent in that ajax request to the server.

Is there any ability to add/send other fields value with the ajax request when performing a remote validation? Basically, I need the ajax request to contain this:

```
verCode:abcd
telegramId:123456789
_jsvalidation:verCode
```

How does this work with Laravels `Required With`, or `same` validation rules? Or is it because they are not a remote validation type this problem doesn't come up?

Thank you for any help you have!

**Comment by torrentalle** [comment]
As you say, this issue is caused because in remote validations only the field to be validated is sent.
The remote validations that depend from other field, doesn't work.
I'm preparing a fix for this issue

**Comment by jonnywilliamson** [comment]
That is brilliant news. Thanks for looking into it!

**Comment by torrentalle** [comment]
fixed in release **v1.3.1**

**Comment by jonnywilliamson** [comment]
Hi,

Just tested this and it's working well! Thanks for implementing this.
